for_developers/product_design.md

```diff
@@ -0,0 +1,25 @@
+# Product Design
+
+## Our Product Mission
+
+1. **Provide a deep learning model training framework that encompasses various tasks solvable by deep learning models.** Our product aims to be convenient for developers, allowing them to easily expand support for tasks and models.
+
+2. **Empower users to seamlessly and intuitively navigate the OpenVINO-based workflow, from model training to inference.** To ensure a user-friendly experience throughout, our product should have user-friendly CLI and Python API entrypoints.
+
+3. **Support Intel devices for accelerating deep learning model training:** We offer best practices for training from scratch or fine-tuning popular model architectures using Intel devices.
```

Comment:
@MarkByun @minjepar 
I think that this should be reviewed by our management line. Please review this carefully. It will be the basis for our future product design decisions.